bin/jobsub_totals

```diff
@@ -24,7 +24,7 @@ totals["T"] = 0
 for line in sys.stdin.readlines():
     print(line, end="")
     words = re.split(r"\s+", line)
-    if len(words) > 4:
+    if len(words) > 5:
```

Comment:
So this certainly works, but do you think it'd be more robust to flag the lines based on a regex rather than a line that's longer than `n` words?  It could even be as simple as looking for lines that start with a `\d+\.\d+@.*` to make sure we're starting with a jobid (I totally threw together that regex - if you do use this suggestion, please check that it actually works).

I only worry that if condor changes its output format in the future, this might come back to bite us.

Reply:
Moved comment to general review comment section